src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -49,6 +50,15 @@ public class ClusterStateMachine : RachisStateMachine
     {
         private const string LocalNodeStateTreeName = "LocalNodeState";
         private static readonly TableSchema ItemsSchema;
+
+        private static readonly TableSchema UniqueItemsSchema;
```

Comment:
I'm not seeing any handling here for sending this over the wire via `InstallSnapshot` ?